### PR TITLE
Add CSV ResultFormatter

### PR DESCRIPTION
**Pull request by tym-xqo**
- Move error message result handler to ResultFormatter base class
- Add local setup.py to Pipfile (`pipenv install -e .`)
- Copy Pipfile.lock into container
- Remove gcsfuse from container

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -48,8 +49,14 @@ class ReportAPI(Resource):
...
     #     let's move this to a separate config
-    query_extension_lookup = {'sql': SQLResultSet, }
-    format_lookup = {'compact': CompactFormatter, 'affix': AffixFormatter}
+    query_extension_lookup = {
+        'sql': SQLResultSet,
+    }
+    format_lookup = {
+        'compact': CompactFormatter,
+        'affix': AffixFormatter,
+        'csv': CsvFormatter,
```
Comment (luzmcosta) [inline app.py:58]: 👍

File: nerium/contrib/formatter/csv.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    # TODO: some graceful error handling, in case of bad input format, etc.
+    def format_results(self):
+        raw = self.result
+        columns = list(raw[0].keys())
+        data = [tuple(row.values()) for row in raw]
+        frame = tablib.Dataset()
+        frame.headers = columns
+        for row in data:
+            frame.append(row)
+        formatted = frame.export('csv')
```
Comment (luzmcosta) [inline nerium/contrib/formatter/csv.py:17]: 👍

File: nerium/main.py [hunk 3]
```diff
@@ -71,6 +71,12 @@ class ResultFormatter(ABC):
     def __init__(self, result):
         self.result = result
 
+    def formatted_results(self):
+        if 'error' in self.result[0].keys():
```
Comment (luzmcosta) [inline nerium/main.py:75]: 💥

**Review by luzmcosta — APPROVED** [review]
✨ 😄 Anywhere I should go to see this running?

File: Pipfile [hunk 4]
```diff
@@ -12,14 +12,15 @@ isort = "*"
...
-Flask-RESTful = "*"
+flask = "*"
+flask-table = "*"
+flask-restful = "*"
 records = "*"
 python-dotenv = "*"
 simplejson = "*"
 gunicorn = "*"
-Frozen-Flask = "*"
+frozen-flask = "*"
```
Comment (henryrizzi) [inline Pipfile]: Are you doing anything with frozen flask?

File: Pipfile [hunk 5]
```diff
@@ -12,14 +12,15 @@ isort = "*"
...
 
-Flask = "*"
-Flask-Table = "*"
-Flask-RESTful = "*"
+flask = "*"
+flask-table = "*"
+flask-restful = "*"
 records = "*"
 python-dotenv = "*"
 simplejson = "*"
```
Comment (henryrizzi) [inline Pipfile]: don't believe you are doing anything with this either (simplejson)

File: Pipfile [hunk 6]
```diff
@@ -12,14 +12,15 @@ isort = "*"
 
 [packages]
 
-Flask = "*"
-Flask-Table = "*"
-Flask-RESTful = "*"
+flask = "*"
+flask-table = "*"
```
Comment (henryrizzi) [inline Pipfile]: I think this also might be a hold-over from before

**Review by henryrizzi — APPROVED** [review]
Besides the comments on the dependencies which seem out of date/not needed this looks good. especially the base class error handling, that's 🔥

File: Pipfile (hunk 6 above)
Comment (tym-xqo) [reply Pipfile]: @henryrizzi You are right about all the obsolete (or aspirational) deps. I can clean that up before I merge.

**Pull request merged**